### DISCUSSION TITLE
Solis Inverter - question about entities

### DISCUSSION THREAD

**Discussion by PistolaPeteUK** [wills106/homeassistant-solax-modbus#439] [Q&A]
Morning

Just a quick question.  I have entities for Battery Voltage, Battery Current, BMS Battery Voltage and BMS Battery Current.
Are the first 2 being measured by the inverter, and the second 2 coming from the batteries (via CANBUS)?

Similarly, are the entities with "meter" in coming from the CT Meter (via the seperate modbus cable to the inverter)

Pete

**Comment by alienatedsec** [comment]
@PistolaPeteUK in short yes, yes, and yes - however, some meter and inverter readings can differ (based on different sources)
